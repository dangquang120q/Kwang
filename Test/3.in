7
33
2748 2593 6487 691 6965 9612 7546 3597 8073 148 7880 8490 5000 3141 3648 7130 7916 540 855 9786 1358 4794 8261 6803 7720 2321 4336 8416 3869 8255 67 6394 3808 
21
5151 1559 3489 9639 8707 1238 2288 9930 7071 6680 1882 674 4479 4785 6182 3850 6386 3925 1404 4395 8221 
10
4929 9615 375 9152 5073 3396 5529 5283 2314 3223 
28
1053 7255 3880 2746 8816 9506 4988 3723 4118 5011 4646 1699 4519 473 6329 3954 5682 5610 6090 6227 6307 2371 8753 9897 3107 6849 6143 6429 
12
4257 5334 4894 1233 5330 2538 2548 5117 135 7933 4602 3007 
66
9954 2475 9550 6235 9411 9692 5691 5063 702 1247 8115 1628 3394 3232 210 3399 6012 9766 2786 8750 7497 3420 2018 6041 9841 7937 2267 4133 1268 1371 988 274 5742 3499 2 5559 3047 3552 1167 2784 5710 4969 4409 2987 9581 8924 9860 1372 9923 2884 9631 3813 4623 6727 2092 6237 4551 941 5627 5585 3605 2835 7620 1206 8510 1385 
68
2800 4737 3343 8013 11 2594 5067 977 1117 6988 5632 2167 7455 86 3734 8455 6473 9382 8147 5987 5792 9004 9908 3862 5349 9281 29 1814 39 2995 6393 1408 820 4702 4596 3737 335 5375 1018 9810 3885 596 431 2865 6325 6802 9905 3827 1836 7718 6219 6366 5206 4938 8963 4513 4145 4570 5468 5905 9186 6797 6345 6485 5892 7604 6515 1979 
