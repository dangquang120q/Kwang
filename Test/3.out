27888
28276
24296
28219
18597
29737
29623
